setup.py

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env python3
-
-from setuptools import setup
-
-if __name__ == "__main__":
-    setup(use_scm_version=True)
```

Comment:
Not sure about removing this one - I thought it is still useful as a back up in some cases?
I forgot when, so might well be fine to remove it by now ...

Reply:
I don't think this is ever needed any more. We can always restore it if someone screams